bedrock/press/templates/press/press-inquiry.html

```diff
@@ -1,106 +1,88 @@
-{% extends "base-resp.html" %}
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-article.html" %}
 
 {% block page_title %}{{_('Press Inquiry Form')}}{% endblock %}
-{% block body_class %}sand{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('press') }}
+{{ css_bundle('press') }}
 {% endblock %}
 
-{% block content %}
-
-<main role="main" id="main-content">
-  <header class="main-header">
-    <h1 class="title-shadow-box">{{_('Press Inquiry')}}</h1>
-  </header>
-
-  <section class="container">
+{% block side_nav %}
+  {% include 'press/includes/resources.html' %}
+{% endblock %}
 
-    {% if not form_success %}
-      <h2 class="secondary-title">{{ _('Please complete and submit the form below with your press inquiry.') }}</h2>
+{% block article %}
+<header>
+  <h1 class="mzp-c-article-title">{{_('Press Inquiry')}}</h1>
+</header>
+{% if not form_success %}
+{% if form.errors %}
+<aside class="mzp-c-notification-bar mzp-t-error">
```

Comment:
Could do with some spacing between this notification and the content below?

![image](https://user-images.githubusercontent.com/400117/69133379-b6294780-0aad-11ea-8950-dfdac7c94546.png)